src/pages/favorites/favorites.html

```diff
@@ -34,7 +36,7 @@
               <ion-col col-auto [style.color]="'#' + route.Color" style="font-size: 120%; font-weight: bold;" >
               {{route?.RouteAbbreviation}}
               </ion-col>
-              <ion-col style="padding-left: 20px;">
+              <ion-col style="padding-left: auto;">
```

Comment:
Maybe using Ionic's [auto columns](http://ionicframework.com/docs/2.2.0/api/components/grid/Grid/#auto-layout-columns) would be a better approach here?  I'm unsure, please tell me otherwise.